data/static/base_info.csv

```diff
@@ -1,4 +1,5 @@
 name_en,first_published,org_type,registry_id,baseline
+Agence Française de Développement,2016-09-30,Other Public Sector,afd,0
```

Comment:
France is a signatory but AFD are not publishing humanitarian data so will not be added to the dashboard. Confirmation received from AFD.